plugins/out_loki/loki.c

```diff
@@ -485,6 +494,38 @@ static int parse_labels(struct flb_loki *ctx)
     return 0;
 }
 
+static int prepare_remove_keys(struct flb_loki *ctx) {
+    int ret;
+    int size;
+    struct mk_list *head;
+    struct flb_slist_entry *entry;
+    struct mk_list *patterns;
+
+    patterns = &ctx->remove_keys_derived;
+
+    /* Add remove keys set in the configuration */
+    if (ctx->remove_keys) {
+        mk_list_foreach(head, ctx->remove_keys) {
+            entry = mk_list_entry(head, struct flb_slist_entry, _head);
+            ret = flb_slist_add_n(patterns, entry->str, flb_sds_len(entry->str));
+            if (ret < 0) {
+                return -1;
+            }
+        }
+    }
+
+    size = mk_list_size(patterns);
+    flb_debug("remove_mpa size: %d", size);
+    if (size > 0) {
+        ctx->remove_mpa = flb_mp_accessor_create(patterns);
```

Comment:
Empty `remove_mpa->ra_list` would just let `flb_mp_accessor_keys_remove()` return `FLB_FALSE`. Should I remove the size check here?